compute/rn/release-information/release-notes-22-06-update2.adoc

```diff
@@ -0,0 +1,90 @@
+== 22.06 Update 2 Release Notes
+
+The following table provides the release details:
+
+[cols="1,4"]
+|===
+|Build
+|22.06.xxx
+
+|Codename
+|Kepler, 22.06 Update 2
+//Tentative date
+|Release date
+|Sep 15, 2022
+
+|Type
+|Maintenance release
+
+|SHA-256 digest
+|
+|===
+
+// Besides hosting the download on the Palo Alto Networks Customer Support Portal, we also support programmatic download (e.g., curl, wget) of the release directly from our CDN:
+//
+// LINK
+
+=== Improvements, Fixes, and Performance Enhancements
+// GithubIssue PCSUP-issue#
+// GH#40449 PCSUP-10632
+* Fixed an issue that caused Defender to incorrectly report the Host OS as SLES15SP1 instead of SLES15.
+
+// GH#40673 #PCSUP-10988  #PCSUP-1940
```

Comment:
@MayaShani Could you please review and confirm that the error was seen all the time, irrespective of the CVE filter!